pepper-apis/src/main/java/org/broadinstitute/ddp/constants/RouteConstants.java

```diff
@@ -89,6 +89,8 @@
                 PathParam.USER_GUID, PathParam.STUDY_GUID, PathParam.INSTANCE_GUID);
         public static final String USER_ACTIVITY_ANSWERS = fmt(BASE + "/user/%s/studies/%s/activities/%s/answers",
                 PathParam.USER_GUID, PathParam.STUDY_GUID, PathParam.INSTANCE_GUID);
+        public static final String USER_SPECIFIC_ANSWER = fmt(BASE + "/user/%s/studies/%s/answers/%s",
```

Comment:
Sorry I'm just realizing that this endpoint is only `/answers/:stableId`. How about we make it `/activities/:instanceGuid/answers/:stableId`? So the idea is that this will be an endpoint for getting a specific answer of an activity instance. This way, our Activities APIs will have [PATCH | PUT | GET] answers. And the endpoint will be general enough to get answers for any instance rather than the latest instance.